tests/test_edit_script.rs

```diff
@@ -90,7 +90,7 @@ fn test_to_arena_empty() {
     assert!(store.to.borrow().is_empty());
     // Generate an edit script.
     let gen_config: Box<EditScriptGenerator<String>> = Box::new(Chawathe96Config::new());
-    let _edit_script_wrapped = gen_config.generate_script(&store); // Panic.
+    gen_config.generate_script(&store); // Panic.
```

Comment:
Of course, this gives a Rust warning `unused std::result::Result which must be used`

Reply:
You can use `.ok()` or `.unwrap()`.